README.md

```diff
@@ -350,6 +376,50 @@ store.on('invalidate', () => {
 });
 ```
 
+### Connecting Store Updates To Widgets
+
+Store data can be connected to widgets within your application using the [Containers & Injectors Pattern](https://github.com/dojo/widget-core#containers--injectors) supported by `@dojo/widget-core`. `@dojo/stores` provides a specialized injector that invalidates store containers on two conditions:
+
+1. The recommended approach is to register `paths` on container creation to ensure invalidation will only occur when state you are interested in changes.
```

Comment:
"will only occur when state you are interested in changes" ?

Reply:
Perhaps should say when state changes for the paths change?